src/selectors-api/selectors-api.json

```diff
@@ -0,0 +1,74 @@
+{
+    "BADGE": ".badge",
+    "BREADCRUMBS": ".breadcrumbs",
+    "BUTTON": {
+        "MAIN": ".btn",
+        "FAKE-BUTTON": ".fake-btn"
+    },
+    "CAROUSEL": ".carousel",
+    "CHECKBOX": ".checkbox",
+    "COMBOBOX": ".combobox",
+    "CTA-BUTTON": ".cta-btn",
+    "DETAILS": ".details",
+    "DRAWER-DIALOG": ".drawer-dialog",
+    "EXPAND-BUTTON": ".expand-btn",
+    "FIELD": ".field",
+    "FILTER-BUTTON": ".filter-button",
+    "FILTER-GROUP": ".filter-group",
+    "FILTER-LINK": ".filter-link",
+    "FILTER-MENU": {
+        "MAIN": ".filter-menu", 
+        "FILTER-MENU-FORM": ".filter-menu-form"
+    },
+    "FILTER-MENU-BUTTON": ".filter-menu-button",
+    "FLOATING-LABEL": ".floating-label",
+    "FULLSCREEN-DIALOG": ".fullscreen-dialog",
+    "ICON": ".icon",
+    "ICON-BTN": {
+        "MAIN": ".icon-btn", 
+        "ICON-LINK": ".icon-link"
+    },
+    "INLINE-NOTICE": ".inline-notice",
+    "LIGHTBOX-DIALOG": ".lightbox-dialog",
+    "LINK": { 
+        "NAV-LINK": ".nav-link", 
+        "ACTION-LINK": ".action-link", 
+        "FAKE-LINK": ".fake-link"
+    },
+    "LISTBOX": {
+        "MAIN": ".listbox",
+        "ACTIVE-ITEM": ".listbox__option[aria-selected='true']"
+    },
+    "LISTBOX-BUTTON": {
+        "MAIN": ".listbox-button",
+        "ACTIVE-ITEM": ".listbox-button__option[aria-selected='true']"
+    },
+    "MENU": { 
+        "MAIN": ".menu", 
+        "FAKE-MENU": ".fake-menu",
+        "ACTIVE-ITEM": ".menu__item[tabindex='0']"
```

Comment:
tabindex=0 would select the currently *focussed* item, is this what they want? To get currently selected item would be a different ARIA based selector. Same applies to MENU-BUTTON below.

Reply:
good catch. I think they want the active one - so yes that would be the selected one. Which aria selector would apply in that case?

Reply:
Actually, I need to expand on that a little more, because for regular menu items there is no selected state, only a focussed state. For checkbox and radio menu items, the attribute is `aria-checked`. 

For fake menus it's different again, and `aria-current` would be the attribute to check for.

Reply:
This will mean shuffling another level of hierarchy under "MAIN" and "FAKE-MENU".

At some point we *might* want to consider flattening out our module structure in Skin, so that fake-menu is moved out of the menu module into it's own top level module. Same goes for the other "fake" stuff. The only downside to that is we lose the benefits of grouped selectors in CSS.

Anyway for now, let's just represent things the way they are currently.

Reply:
`aria-selected=true` seems like

Reply:
fixed this

Reply:
actually still need to separate different menu item types...